web/src/engine/websites/ComicDays_e2e.ts

```diff
@@ -19,4 +20,48 @@ new TestFixture({
         size: 2_259_808,
         type: 'image/png'
     }
+}).AssertWebsite();
+
+// Case : volumes
+new TestFixture({
+    plugin: {
+        id: 'comicdays',
+        title: 'コミックDAYS (Comic Days)'
+    },
+    container: {
+        url: 'https://comic-days.com/volume/2550689798642753315',
+        id: '/volume/2550689798642753315',
+        title: 'ＪＪＭ 女子柔道部物語 社会人編',
+    }, /* Paid content
```

Comment:
Arn't there no free chapters?

Reply:
No free **volumes**, no. No website gives entire free volumes to test. They sell volumes after all.